os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_central/ble_tizenrt_client.c

```diff
@@ -139,9 +312,22 @@ trble_result_e rtw_ble_client_start_scan_with_filter(trble_scan_filter* scan_par
     } else {
         debug_print("\r\n[%s] set scan info success", __FUNCTION__);
     }
-    
-    if(ble_tizenrt_client_send_msg(BLE_TIZENRT_START_SCAN, NULL) == false)
+
+    uint8_t *scan_filter_policy = os_mem_alloc(0, sizeof(uint8_t));
+    if(scan_filter_policy == NULL)
+    {
+        debug_print("\n[%s] Memory allocation failed", __FUNCTION__);
+        return TRBLE_FAIL;
+    }
+    *scan_filter_policy = GAP_SCAN_FILTER_ANY;
```

Comment:
How about like below?
```
if (whitelist_enable == true) {
    *scan_filter_policy = GAP_SCAN_FILTER_WHITE_LIST;
} else {
    *scan_filter_policy = GAP_SCAN_FILTER_ANY;
}
```
Then we don't set as `GAP_SCAN_FILTER_ANY` if whitelist is enabled.
```

Reply:
Modified in latest commit.
Thanks